library/Fission/Web/Swagger.hs

```diff
@@ -68,6 +69,10 @@ ping :: Swagger -> Swagger
 ping = makeDocs (Proxy @Web.PingRoute)
   ["Ping" & description ?~ "Check for liveness"]
 
+dns :: Swagger -> Swagger
+dns = makeDocs (Proxy @Web.DNSRoute)
+  ["DNS" & description ?~ "Interact with DNS on AWS Route53"]
```

Comment:
🤔 We probably don't need to be this specific. Let's say we later switch provider. "DNS management" or something may be sufficient.